tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"extends": "@vue/tsconfig/tsconfig.json",
+	"include": ["./src/**/*.ts"],
+	"compilerOptions": {
+		"types": ["cypress", "node", "dockerode"],
+		"allowSyntheticDefaultImports": true,
+		"moduleResolution": "node",
+		"target": "ESNext",
+		"module": "esnext",
+		"declaration": true,
+		"strict": true,
+		"noImplicitAny": false,
+		"resolveJsonModule": true
+	},
+	"ts-node": {
+		// these options are overrides used only by ts-node
+		// same as our --compilerOptions flag and our TS_NODE_COMPILER_OPTIONS environment variable
```

Comment:
Comments in JSON?

Reply:
Looks like yeah. Don't seems to cause any issue :see_no_evil: